models/modelArticles.js

```diff
@@ -15,4 +15,37 @@ function retrieveArticleDataById(article_id) {
     });
 }
 
-module.exports = { retrieveArticleDataById };
+function retrieveAllArticleData(sort_by = "created_at", order = "DESC") {
+  const validOrders = ["ASC", "DESC"];
+
+  if (!validOrders.includes(order)) {
```

Comment:
You didn't need to implement this at this stage, but if you were going to then you should have tested that this 400 error handler works